src/lib/y2network/autoyast_routing.rb

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/routing_helpers"
+
+Yast.import "Lan"
+
+module Y2Network
+  # This class is responsible of importing and exporting AutoYast routing
+  # configuration
+  class AutoyastRouting
+    include Y2Network::RoutingHelpers
```

Comment:
To be honest, always I see *Helpers object it rings bell in my head like it is next utils, complex or routines include that holds mess that does not fit elsewhere. I prefer more focused mixins name.

Reply:
:) I though it was focused enough, but sure, we could think in other options for sharing these common methods